app/gui/src/dashboard/utilities/math.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @file Math utilities.
+ */
+
+/**
+ * Clamp a value between a minimum and maximum.
+ * @param value - The value to clamp.
+ * @param min - The minimum value.
+ * @param max - The maximum value.
+ * @returns The clamped value.
+ */
+export function clamp(value: number, min: number, max: number) {
+  return Math.max(min, Math.min(value, max))
```

Comment:
i prefer `min, max, value` (well, anything that puts `min` and `max` together. although in this case i suppose it makes sense to have `value` in the middle

Reply:
Yeah, I prefer `(min, value, max)` but it's such a discussible thing, because of `curring`

Though even with `(min, value, max)` it's possible, but tedious